packages/lab/src/list-next/ListNextContext.tsx

```diff
@@ -0,0 +1,18 @@
+import { createContext } from "@salt-ds/core";
+import { SyntheticEvent, useContext } from "react";
+
+export interface ListItemNextContextValue {
+  disabled: boolean;
```

Comment:
I assume this should be optional
```suggestion
  disabled?: boolean;
```